### PR TITLE
Initial subscriptions docs

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: extensions/amp-subscriptions-google/amp-subscriptions-google.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-subscriptions-google"></a> `amp-subscriptions-google`
+
+<table>
+  <tr>
+    <td class="col-fourty"><strong>Description</strong></td>
+    <td>Dynamically injects ads into an AMP page by
```
Comment (prateekbh) [inline extensions/amp-subscriptions-google/amp-subscriptions-google.md]: Dynamically injects ads??? 
I guess this one is a WIP or something?

File: extensions/amp-subscriptions/amp-subscriptions.md [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+    ...
+  ]
+}
+</script>
+```
+
+The properties in the "local" service are:
+ - "authorizationUrl" - the authorization endpoint URL.
+ - "pingbackUrl" - the pingback endpoint URL.
+ - "actions" - a named map of actions. At a minimum there must be two actions specified: "login" and "subscribe".
```
Comment (prateekbh) [inline extensions/amp-subscriptions/amp-subscriptions.md]: a named map of "urls" instead may be?

File: extensions/amp-subscriptions/amp-subscriptions.md [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-subscriptions"></a> `amp-subscriptions`
+
+<table>
+  <tr>
+    <td class="col-fourty"><strong>Description</strong></td>
+    <td>Dynamically injects ads into an AMP page by
```
Comment (prateekbh) [inline extensions/amp-subscriptions/amp-subscriptions.md]: Same as first one

File: extensions/amp-subscriptions-google/amp-subscriptions-google.md (hunk 1 above)
Comment (dvoytenko) [reply extensions/amp-subscriptions-google/amp-subscriptions-google.md]: Fixed.

File: extensions/amp-subscriptions/amp-subscriptions.md (hunk 2 above)
Comment (dvoytenko) [reply extensions/amp-subscriptions/amp-subscriptions.md]: Done

File: extensions/amp-subscriptions/amp-subscriptions.md (hunk 3 above)
Comment (dvoytenko) [reply extensions/amp-subscriptions/amp-subscriptions.md]: Done

**Pull request merged**